content/blog/2020/01/2020-01-10-fosdem-is-coming.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "FOSDEM 2020 is coming"
+tags:
+- fosdem
+- events
+- community
+- outreach-programs
+author: olblak
+opengraph:
+  image: /images/post-images/jenkins-atomium.png
+---
+
+image::/images/post-images/fosdem[FOSDEM, role=center]
+
+link:https://fosdem.org/2020/[FOSDEM] 2020 is coming and with it, a lot of great folks come in town.  
+It’s always a great moment to meet Jenkins community members, share stories and get inspired.
+I hope that this year will be as great as it always been and for that, we organize a few things
+
+== Things we'll do
+
+During the whole event, we'll be virtually on the link:https://gitter.im/jenkinsci/fosdem[Gitter]
+
+On the Thursday 30 of January, there will be two workshops one about link:https://www.eventbrite.com/e/jenkins-pipeline-fundamentals-training-tickets-87080214265[Jenkins Pipelines] lead by Mark Waite, and a second one about link:https://www.eventbrite.com/e/workshop-cloud-native-kubernetes-first-serverless-continuous-delivery-with-jenkins-x-kubernetes-and-tickets-87082627483[JenkinsX] by Viktor Farcic.
+
+On the Friday 31 of January, the Jenkins project will hold a link:https://www.meetup.com/jenkinsmeetup/events/267684785/[Contributor Summit] where we invite active contributors and those who are interested in working on foundation projects, e.g. key architecture changes and projects (UX, JCasC, Cloud native Jenkins, etc.), governance, infrastructure. There will be no user-focused topics (no presentations, no trainings, etc.) but we will focus on defining key priorities for the project, building a roadmap and resolving issues we have in the project at the moment.
+We'll end up the day with our now traditional Orval and flemish beef stew at http://roydespagne.be/fr/index.php[Le Roy d'Espagne] 
+
+Finally the FOSDEM, the 01 and 02 of February, we’ll all be at FOSDEM. So come and say "hi" at the Jenkins/JenkinsX link:https://fosdem.org/2020/stands/[stand], inside the link:https://fosdem.org/2020/schedule/track/continuous_integration_and_continuous_deployment/[CICD Devroom].
+
+Or just come and share beers
+
+Cheers
+
+image::/images/post-images/jenkins-atomium.png[jenkins-atomium, role=center]
```

Comment:
@olblak was this image supposed to be in the `/images/post-images/fosdem` folder?

Reply:
Why should it be?
It's not specific to the Fosdem but just the atomium jenkins logo